python/ppml/example/pytorch_nn_lr/pytorch_nn_lr_1.py

```diff
@@ -83,11 +87,9 @@ def run_client(load_model):
         
         server_model = ServerModel()
         ppl = Estimator.from_torch(client_model=model,
-                                   client_id='1',
                                    loss_fn=loss_fn,
                                    optimizer_cls=torch.optim.SGD,
-                                   optimizer_args={'lr':1e-5},
-                                   target='localhost:8980',
+                                   optimizer_args={'lr':1e-4},
                                    server_model=server_model,
                                    server_model_path='/tmp/pytorch_server_model',
                                    client_model_path='/tmp/pytorch_client_model_1.pt')
```

Comment:
save after `fit`?

Reply:
Estimator would automatically save after each epoch of `fit` like Orca Estimator.